app/(waterLevel)/water-level/sensor/[sensorId]/[startTime]/[endTime]/[datum]/page.tsx

```diff
@@ -29,27 +29,46 @@ export default function SensorIdPage({ params }) {
   }, [data])
 
   return (
-    <Row>
-      <Col style={{ width: "20vw" }}>
-        <div style={{ width: "100%" }}>
+    <div>
+      {window.innerWidth < 576 ? (
+        <Col>
```

Comment:
This is pretty ugly and is a very manual way to set the order of the components for mobile vs non-mobile. Because its not a perfect grid (i.e. the bottoms/tops don't align in side/side view) the sidebar components have to be wrapped in the large view but not in the mobile view since the `platformInfo` and map components need be be separate to add the graph between. 
Feedback absolutely welcome @abkfenris  but not sure how much more its worth fudging around with it.

Reply:
I think this could be done with [Bootstrap/Reactstrap breakpoint classes](https://getbootstrap.com/docs/5.3/layout/grid/#css). The other option might be `useMeasure` or one of the other utilities in `react-use`. https://streamich.github.io/react-use/?path=/story/sensors-createbreakpoint--docs

Reply:
Thanks for sending these over. I looked at them and they don't seem to solve the specific problem of what we have going on, that is, conditionally rendering a container in order to avoid conditionally rendering the contents twice. I did add `useBreakpoint` to avoid the use of calling `window.innerWidth`.